packages/tradershub/src/components/SentEmailContent/SentEmailContent.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { FC, Fragment, useEffect, useState } from 'react';
+import { useCountdown } from 'usehooks-ts';
+import { useActiveTradingAccount, useSettings, useVerifyEmail } from '@deriv/api';
+import { Button, useBreakpoint } from '@deriv/quill-design';
+import { PlatformDetails } from '../../features/cfd/constants';
+import ChangePassword from '../../public/images/change-password-email.svg';
+import { TPlatforms } from '../../types';
+import { platformPasswordResetRedirectLink } from '../../utils/cfd';
+import { ActionScreen } from '../ActionScreen';
+
+type TProps = {
+    description?: string;
+    isInvestorPassword?: boolean;
+    platform?: TPlatforms.All;
+};
+
+const SentEmailContent: FC<TProps> = ({ description, isInvestorPassword = false, platform }) => {
+    const [shouldShowResendEmailReasons, setShouldShowResendEmailReasons] = useState(false);
+    const [hasCountdownStarted, setHasCountdownStarted] = useState(false);
+    const { data } = useSettings();
+    const { mutate: verifyEmail } = useVerifyEmail();
+    const { isMobile } = useBreakpoint();
+    const mt5Platform = PlatformDetails.mt5.platform;
+    const title = PlatformDetails[platform ?? mt5Platform].title;
+    const titleSize = 'md';
+    const descriptionSize = 'sm';
+    const emailLinkSize = isMobile ? 'lg' : 'md';
+    const [count, { resetCountdown, startCountdown }] = useCountdown({
+        countStart: 60,
+        intervalMs: 1000,
+    });
+
+    useEffect(() => {
+        if (count === 0) setHasCountdownStarted(false);
+    }, [count]);
+
+    const { data: activeTrading } = useActiveTradingAccount();
+
+    return (
+        <div className='w-full lg:w-[400px] inline-flex p-1600 flex-col justify-center items-center gap-1200 rounded-400 bg-system-light-primary-background'>
+            <ActionScreen
+                description={description ?? `Please click on the link in the email to change your ${title} password.`}
+                descriptionSize={descriptionSize}
+                icon={<ChangePassword />}
+                renderButtons={() =>
```

Comment:
Please move this condition outside of the return and just pass the function here